### DISCUSSION TITLE
希望中中枢网关的虚拟事件，增加【事件数据】参数

### DISCUSSION THREAD

**Discussion by xdatoutou** [XiaoMi/ha_xiaomi_home#99] [Ideas]
现在中枢网关产生虚拟事件只能传递【事件名称】参数，有时候希望从HA传输一些【事件数据】到 中枢极客板，与极客板的变量进行互动，现在无法实现。
比如HA判断家中有3个人，希望能通知极客板。
事件名称:家中人数变化
事件数据:3

**Comment by JimmyKmi** [comment]
参考：https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/5lit5p6i572r5ywz6jma5ouf5lql5lu26kem5yr5yk5omn6kgm-BBcVquoizJ

**Reply by xdatoutou** [reply]
这个只支持事件名称，我希望是事件名称 ➕事件数据
比如事件名称：家中有人
时间数据：3

这个功能应该要更新中枢软件，不然极客版无法接收数据参数

**Comment by luanpao888** [comment]
强烈建议增加参数传递。定长报文都可以接受
